openfisca_core/periods.py

```diff
@@ -358,6 +358,41 @@ def __str__(self):
             if fragment is not None
             )
 
+    def __cmp__(self, other):
```

Comment:
@When do we need to compare periods based on their size ?

If we need this, we must document it : defining an order on periods is nothing but obvious (here `period('2014') > period('2015-01')`, which some users may not expect.`